extensions/postgres/transactional.go

```diff
@@ -69,6 +70,17 @@ func (d dbTx) SelectAsyncStateExecutionForUpdate(
 	return &row, err
 }
 
+const selectRunningAsyncStateExecutionCountQuery = `SELECT 
```

Comment:
I think the pending sequence map in the process execution can provide the information